pkg/apiclient/apiclient.go

```diff
@@ -417,7 +418,11 @@ func (c *client) refreshAuthToken(localCfg *localconfig.LocalConfig, ctxName, co
 		return err
 	}
 	c.AuthToken = rawIDToken
-	c.RefreshToken = refreshToken
+	// When the oidc provider returns an empty refresh token, use the existing refresh token
```

Comment:
Found this problem while testing it. Our identity provider doesn't issue a new refresh token and the existing implementation would just delete it from the local config.

> The authorization server MAY issue a new refresh token, in which case
   the client MUST discard the old refresh token and replace it with the
   new refresh token.  The authorization server MAY revoke the old
   refresh token after issuing a new refresh token to the client.  If a
   new refresh token is issued, the refresh token scope MUST be
   identical to that of the refresh token included by the client in the
   request.

from https://datatracker.ietf.org/doc/html/rfc6749#section-6 thank you @fabiolanfranchi for pointing it out